modules/core/src/main/kotlin/datamaintain/core/step/check/rules/implementations/ExecutedScriptsNotRemovedCheck.kt

```diff
@@ -0,0 +1,36 @@
+package datamaintain.core.step.check.rules.implementations
+
+import datamaintain.core.script.ExecutedScript
+import datamaintain.core.script.ScriptWithContent
+import datamaintain.core.step.check.rules.ScriptType
+import datamaintain.core.step.check.rules.contracts.FullContextCheckRule
+import java.lang.IllegalStateException
+
+class ExecutedScriptsNotRemovedCheck(
+        executedScripts: Sequence<ExecutedScript>
+) : FullContextCheckRule(executedScripts) {
+    override fun check(scripts: Sequence<ScriptWithContent>) {
+        val executedScriptChecksumsNotFoundInSortedScripts = executedScripts
```

Comment:
Since the script type used by this rule is ```SCANNED_SCRIPT```, this variable is not named properly. It should be ```executedScriptChecksumsNotFoundInScannedScripts```